### ISSUE TITLE
Overriding `before`, `handler` and `after` for `Actions.new` isn't working.

### ISSUE THREAD

**Issue by andresumanzor** [SoftwareBrothers/admin-bro#450]
Hi team,

Hope you are all safe and doing amazing!

**Describe the bug**
Replacing either the `before`, `handler` and `after` functions for resource.options.actions.new isn't working. Eg, the following isn't working either: https://adminbro.com/admin-bro_src_backend_actions_action.interface.ts.html #line475

**Installed libraries and their versions**

* admin-bro@2.0.1
* admin-bro-expressjs@2.0.5

**Expected behavior**
It to override the "New" action `before` and `after` functions.

**AdminBroOptions with schema**
```
{
  resources: [
    {
      resource: users,
      options: {
        parent: menu.mainTables,
        actions: {
          new: {
            actionType: "resource",
            before: async (request) => {
              console.log({ request });
              if (request.payload.password) {
                request.payload = {
                  ...request.payload,
                  encryptedPassword: await bcrypt.hash(
                    request.payload.password,
                    10
                  ),
                  password: undefined,
                };
              }
              return request;
            },
          },
          edit: {
            actionType: "record",
            before: beforeEditHandler,
            handler: editHandler,
            after: afterEditHandler,
          },
        },
      },
    },
  ],
  branding: {
    logo: `urlToLogo`,
    favicon:
      "urlToFavicon",
    companyName: "CompanyName",
  },
}
```

If you do the above, not even `request` is being logged.

Many thanks in advance :)

**Comment by wojtek-krysiak** [comment]
hmm - it looks correct. But does any other options work (branding for instance)?

Maybe take a look at the example from jscasts.tv: https://github.com/JSCasts-episodes/ep18-admin-bro-startup-part3 - maybe it will help you debug this

**Comment by andresumanzor** [comment]
Hey @wojtek-krysiak,

I hope you and your beloved ones are doing ok.

Everything else but overriding the `before`, `handler`, and `after` for the `new` resource action methods is working just fine.

What I ended up doing is creating a completely custom resource action that would help me mimic this one so for me to be able to just customize everything else I needed to within it.

Besides that, I rolled back to admin-bro `v1.6.0` alongside all of its dependencies because `2.0.1` was really not working well for custom components and was randomly crashing at some points.

However, I do really appreciate all of the efforts you have put into this project and hope you can catch up with these and all of the other common bugs with the newer versions as it will be the bomb.

Hope you have an amazing weekend and keep on rocking 🤘

**Comment by wojtek-krysiak** [comment]
thanks. Maybe you can give v2.3.1 a try :)

**Comment by andresumanzor** [comment]
Hi @wojtek-krysiak,

Thanks for your reply.

For sure, I will do so asap and let you know how it goes. If this can't be reproduced on v2.3.1 then, I think it is ok if you close it and if anything then, I will open a new issue.

Hope you have an amazing week 👍
